Makefile

```diff
@@ -35,6 +35,9 @@ test-integration:
 test-e2e:
 	# Command to run e2e tests goes here
 
+coverage-report:	test-unit
+	coverage html
```

Comment:
where does this expect to pick up the coverage report from and does it align w/ where test-unit will put them(especially since test-unit picks the output dir from an env variable)?  I'd prefer to see this be explicit.

Reply:
I really think that coverage database is still created in project root directory. It does not depend on `ARTIFACT_DIR` AFAIK, but let me check

Reply:
yes, the `ARTIFACT_DIR` does not change where the coverage database is stored.

Reply:
looks like `.coverage` is already in our .gitignore, but can you add whatever file(s) this generates?

Reply:
yes, sure, nice idea

Reply:
@bparees updated